python/CCLE_data.py

```diff
@@ -38,29 +42,123 @@ def common_celllines(a,b):
 
 #Used to create correlation matrix
 def Matrix(gene_list, histone_list, data1, data2):
```

Comment:
name your functions better. matrix does not tell me what this function is doing.